src/App.tsx

```diff
@@ -5,6 +5,10 @@ import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
   const [search, setSearch] = useState('');
+  const moviesForVisible = moviesFromServer.filter(movie => {
+    return movie.title.toLowerCase().includes(search.toLowerCase())
+        || movie.description.toLowerCase().includes(search.toLowerCase());
```

Comment:
*optional 

You can make a function which will do `.toLowerCase().includes(search.toLowerCase());` operation for every parameter, you will push in this function.